src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForObservedDataFolder.cs

```diff
@@ -41,6 +50,15 @@ public override IEnumerable<IMenuBarItem> AllMenuItems()
          if (groupMenu.AllItems().Any())
             yield return groupMenu;
 
+         if (_treeNode.HasChildren)
+         {
+            var colorGroupingButton = CreateMenuCheckButton.WithCaption(Captions.ColorGroupObservedDataContextMenu);
```

Comment:
even this guy could be a menu in core. We have generic menu definitions (see PI etc..)

Reply:
@georgeDaskalakis You did not change this! This needs to be written like all other menus
Please update
var colorGroupingButton = CreateMenuCheckButton
.WithCaption(Captions.ColorGroupObservedDataContextMenu)
.WithChecked*(xx
.WithCheckedAction

I did not verify before merging.